application/logs/log-2019-10-17.php

```diff
@@ -0,0 +1,82 @@
+ERROR - 2019-10-17 14:07:59 --> blacklist_number/
```

Comment:
I hope this file is not required.

Reply:
now i delete this file and set the config log to 0